app/views/items/show.html.erb

```diff
@@ -1,16 +1,21 @@
 <% content_for :sidebar do %>
   <%= link_to 'Return to Listing', items_path, :class => 'btn' %></br></br>
   <%= link_to 'Edit Item', edit_item_path(@item), :class => 'btn btn-primary' unless cannot? :edit, @item  %></br></br>
+  <%= link_to 'Add Repair', new_item_repair_path(@item), :class => 'btn btn-primary' unless cannot? :edit, @item  %></br></br>
+  <%= link_to 'Add Inspection', new_item_inspection_path(@item), :class => 'btn btn-primary' unless cannot? :edit, @item  %></br></br>
```

Comment:
Could you also shorten these 2 lines, and the long lines in `spec/features/person_edit_spec.rb`?